shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/openapi/OpenApiReaderImplementation.java

```diff
@@ -29,8 +29,11 @@ public static OpenApiReaderImplementation getInstance() {
     public String openAsString(String fileName) throws UnableToReadOpenApiSpecificationException {
         InputStream stream = getClass().getClassLoader().getResourceAsStream(fileName);
         try {
-            return new String(
-                    Objects.requireNonNull(stream).readAllBytes(), StandardCharsets.UTF_8);
+            var out =
+                    new String(
+                            Objects.requireNonNull(stream).readAllBytes(), StandardCharsets.UTF_8);
+            stream.close();
```

Comment:
Can you use a try/catch managed scope to close the InputStream? Something like this: `try (InputStream stream = getClass().getClassLoader().getResourceAsStream(fileName)) {`. Also, now that you're here, it may be a good idea to pass the utf8 charset as an argument

Reply:
Ok, updated it